CONTRIBUTING.md

```diff
@@ -67,21 +75,36 @@ git submodule update --init --recursive
 apt install libxml2-utils
 ```
 
-#### Launch all tests
+#### Check outputs manually
 
+To validate the output NeTEx obtained it is possible to use xmllint:
 ```sh
-# Run all the tests of `transit_model` in the entire repository,
-# activating all features, including `xmllint`
-cargo test --workspace --all-features
+xmllint --noout --nonet --huge --schema /path/to/NeTEx/xsd/NeTEx_publication.xsd your_file.xml
 ```
 
-#### Hand-check outputs
+#### Launch all tests
 
-To validate the output NeTEx obtained it is possible to use xmllint:
 ```sh
-xmllint --noout --nonet --huge --schema /path/to/NeTEx/xsd/NeTEx_publication.xsd your_file.xml
+# Run all the tests of `transit_model` in the entire repository,
+# without fixtures, then activating all features, including `xmllint`
+# to make sure that both works
+cargo test --workspace ; cargo test --workspace --all-features
```

Comment:
```suggestion
cargo test --workspace && cargo test --workspace --all-features
```
Maybe what you did was intended but with all the output on `stdout`, if the first command fails, it will be buried in the logs of the second command.

Reply:
OK, true.
Maybe then we should start with --all-features if it's longer and has more chances to crash, to minimize waiting time on the rebuild?